alerting/src/main/kotlin/org/opensearch/alerting/settings/LegacyOpenDistroAlertingSettings.kt

```diff
@@ -26,6 +26,12 @@ class LegacyOpenDistroAlertingSettings {
             Setting.Property.NodeScope, Setting.Property.Dynamic, Setting.Property.Deprecated
         )
 
+        val ALERTING_TRIGGER_MAX_ACTIONS = Setting.intSetting(
```

Comment:
This can be removed since the setting is only being introduced in an OpenSearch version.

Reply:
> This can be removed since the setting is only being introduced in an OpenSearch version.

Got it!